composer.json

```diff
@@ -21,13 +22,20 @@
 	},
 	"require": {
 		"php": ">=7.1",
+		"bjeavons/zxcvbn-php": "^0.4.0",
 		"altis/browser-security": "^1.0.0",
-		"humanmade/stream": "3.3.0",
-		"humanmade/two-factor": "0.2",
+		"humanmade/php-basic-auth": "dev-enhancement-add-action-to-env-check",
 		"humanmade/require-login": "1.0.1",
-		"bjeavons/zxcvbn-php": "^0.4.0"
+		"humanmade/stream": "3.3.0",
+		"humanmade/two-factor": "0.2"
 	},
 	"extra": {
-		"altis": {}
+		"altis": {},
+		"installer-paths": {
```

Comment:
Chatted with @goldenapples a bit and mentioned you were having issues with this. It's worth highlighting the difference between dependencies and the root composer (project) `composer.json`. If you look at the `composer.json` schema reference it shows some properties are "root-only".

https://getcomposer.org/doc/04-schema.md

Unfortunately it doesn't go as far as to make it clear the `installer-paths` extra data is not supported in dependencies themselves so you can remove this part.

Reply:
You may have hit issues locally with composer caches etc when running `composer update` after making changes. The nuclear option is:

```
composer clear-cache
rm -rf composer.lock vendor/
composer update
```